components/automate-ui/src/app/modules/infra-proxy/infra-role-details/infra-role-details.component.ts

```diff
@@ -106,7 +106,7 @@ export class InfraRoleDetailsComponent implements OnInit, OnDestroy {
       this.expandedList = role.expanded_run_list;
       this.runList = this.role.run_list;
       this.idList = [];
-      this.attributes = new RoleAttributes(this.role);
+      // this.attributes = new RoleAttributes(this.role);
```

Comment:
why we are commenting this, can u explain

Reply:
i have updated the changes